stored_requests/backends/db_fetcher/fetcher.go

```diff
@@ -93,6 +99,51 @@ func (fetcher *dbFetcher) FetchRequests(ctx context.Context, requestIDs []string
 	return storedRequestData, storedImpData, errs
 }
 
+func (fetcher *dbFetcher) FetchResponses(ctx context.Context, ids []string) (data map[string]json.RawMessage, errs []error) {
+	if len(ids) < 1 {
+		return nil, nil
+	}
+
+	query := fetcher.responseQueryMaker(len(ids))
+	idInterfaces := make([]interface{}, len(ids))
+	for i := 0; i < len(ids); i++ {
+		idInterfaces[i] = ids[i]
+	}
+
+	rows, err := fetcher.db.QueryContext(ctx, query, idInterfaces...)
+	if err != nil {
+		fmt.Println(err.Error(), query)
```

Comment:
Do we need to be printing the error to console, instead of just returning this error?

Reply:
Yes, updated :)